### ISSUE TITLE
Is it possible G.711 codec for audio stream ? 

### ISSUE THREAD

**Issue by Midhun9638** [fyhertz/libstreaming#282]
I need to stream audio on wowza webRTC as it not support AAC. can anyone help please?

**Comment by Programmerpk** [comment]
Where the Problem arise to u

**Comment by Programmerpk** [comment]
but this library support only ACC and ARM you should use another one

**Comment by Midhun9638** [comment]
Yes I already using AAC but now need g.711 codec for webRCT support on wawza
